addon/adapter.js

```diff
@@ -156,13 +156,24 @@ export default DS.Adapter.extend({
 
     serializer.serializeIntoHash(data, type, snapshot);
 
+    let payload = { id: data['id'] };
+    Object.keys(snapshot.changedAttributes()).forEach(key => {
+      let payloadKey = serializer.keyForAttribute(key);
+      payload[payloadKey] = data[payloadKey];
+    });
+
+    snapshot.eachRelationship((key, {kind, options}) => {
```

Comment:
Unlikely, the previous iteration was through attributes. This iteration is throuhg relationships.